src/components/Comments.jsx

```diff
@@ -0,0 +1,51 @@
+import "../styling/Comments.css";
+import { useEffect, useState } from "react";
+import { getComments } from "../utils.js";
+
+export default function Comments() {
+  const [commentsById, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getComments(2).then(({ comments }) => {
```

Comment:
I think this is your culprit, you're always getting comments for id 2

Reply:
Thank you, this has now been changed.